.github/ISSUE_TEMPLATE/1.bug-and-feature.md

```diff
@@ -1,3 +1,8 @@
+---
+name: Bug report or feature request
```

Comment:
What do you think about separating these into two templates?

Reply:
What would be the difference?

Reply:
Something like this.